server/src/resolvers/mapping/utils.ts

```diff
@@ -25,6 +26,35 @@ export const clean = (entry: any): any => {
   return ret
 }
 
+export const checkAuthors = async (
+  prismaClient: PrismaClient,
+  resources: any[],
+) => {
+  const authorMails: string[] = resources.reduce(
+    (authors, resource) => [
+      ...authors,
+      ...resource.attributes.reduce(
+        (resourceAuthors: string[], attribute: AttributeWithComments) => [
+          ...resourceAuthors,
+          ...attribute.comments.map(comment => comment.author.email),
+        ],
+        [],
+      ),
+    ],
+    [],
+  )
+  const uniqueMails = authorMails.filter(
+    (mail, ind) => authorMails.indexOf(mail) === ind,
+  )
+  const allUsers = await prismaClient.user.findMany({
+    where: { email: { in: uniqueMails } },
+  })
+  const existingMails = allUsers.map(user => user.email)
+  const missingUsers = uniqueMails.filter(mail => !existingMails.includes(mail))
+  if (missingUsers.length > 0)
+    throw Error(`missing users ${missingUsers.join(', ')}`)
```

Comment:
I liked your previous error message "trying to import a mapping with unexisting comment author ${missingUsers.join(', '}"